includes/rest/class-collection.php

```diff
@@ -186,7 +186,7 @@ public static function featured_get( $request ) {
 	 */
 	public static function moderators_get( $request ) {
 		$response = array(
-			'@context'     => Activity::CONTEXT,
+			'@context'     => Activity::JSON_LD_CONTEXT,
```

Comment:
With your changes, I think `Activity::JSON_LD_CONTEXT` is no longer enough, it has to be the Object context now and this can be a bit complicated?!?